configure.ac

```diff
@@ -156,6 +156,7 @@ else
     (cd src/third_party && tar -xzf ../../${zstd_archive})
     rm -rf src/third_party/zstd
     mv src/third_party/zstd-${zstd_version} src/third_party/zstd
+    rm -f ${zstd_archive}
```

Comment:
+1 for keeping the repo clean! However, not removing the downloaded archives is intentional – that way you don't have to download them again when rerunning the configure script (see the `if` check 16 lines above).

What do you think about adding the archives to `.gitignore` instead?

Reply:
Fine by me :)